moment.easter.d.ts

```diff
@@ -0,0 +1,10 @@
+import * as moment from 'moment';
+
+declare module 'moment' {
+    interface Moment {
+        easter(year?: number): Moment;
```

Comment:
Are you sure there's a year argument?
It's been a very long time since I used this so I'm not sure

Reply:
Yes, there are two basic ways to use a function, according to README.
moment.easter(2014).format('YYYY/MM/DD');
moment({y: 2015}).easter().format('YYYY/MM/DD');

Reply:
Am I correct in assuming that the first case corresponds to the `export` below and the second to the `interface` above?

Reply:
Yes, you're right.

Reply:
In that case, this method doesn't have a `year` argument

Reply:
OHH, sorry... I tried to leave it as an optional arg, but the implementation doesn't follow that.
I'm push a new commit to fix this...